examples/BalanceTxConstraints.purs

```diff
@@ -132,9 +138,16 @@ contract (ContractParams p) = do
       $ head
       <$> (withKeyWallet p.bobKeyWallet getWalletAddressesWithNetworkTag)
 
+  bobsCollateralArray <- withKeyWallet p.bobKeyWallet do
+    fold <$> getWalletCollateral
+  let
+    bobsCollateral =
+      Map.fromFoldable $ bobsCollateralArray <#> unwrap >>>
+        \{ input, output } -> Tuple input output
+
   nonSpendableOref <-
     liftedM "Failed to get utxos at Bob's address"
-      (Set.findMin <<< Map.keys <$> utxosAt bobAddress)
+      (Set.findMin <<< Map.keys <$> utxosAt aliceAddress)
```

Comment:
We should check that none of the Alice's utxos were spent

Reply:
This is a trivial property